File: katas/01-ean_validator/gabriel.py
def validate_ean(code):
    checksum = 0
    for index, digit in enumerate(code[:-1]):
        index += 1
        int_digit = int(digit)
        checksum += int_digit        
        if index % 2 == 0:
            checksum += int_digit*2
    return str(10 - (checksum % 10))[-1] == code[-1]

File: katas/01-ean_validator/gabriel.js
function validateEAN(eanCode) {
  let checksum = 0;
  for (let i=0; i<eanCode.length-1; i++){
    checksum += Number(eanCode[i]);
    if (i%2!=0){
      checksum += Number(eanCode[i])*2;
    }
  }
  checksum = (10 - (checksum % 10)).toString();
  return eanCode[eanCode.length-1] == checksum[checksum.length-1]
}

